tests/test_getpoly.f90

```diff
@@ -0,0 +1,37 @@
+! test getpoly.F90
```

Comment:
Something like:

This is a test for the NCEPLIBS-g2 project. It tests the getpoly subroutine.

Reply:
We can get this in any future PR, no problem.

IMO seems redundant.